python/cuml/dask/ensemble/randomforestcommon.py

```diff
@@ -0,0 +1,173 @@
+import math
+
+from cuml.dask.common import raise_exception_from_futures
+from cuml.dask.common.input_utils import DistributedDataHandler, \
+    concatenate
+
+from dask.distributed import wait
+
+from uuid import uuid1
+
+
+class BaseRandomForestModel(object):

```

Comment:
I really like the consolidation here! The `decomposition` and `linear_models` packages both name this file `base.py`, though. I think it's good to maintain consistency throughout the codebase even for small details like this.